packages/web-twig/src/Resources/components/Field/ValidationText.twig

```diff
@@ -2,41 +2,57 @@
 {%- set props = props | default([]) -%}
 {%- set _className = props.className -%}
 {%- set _elementType = props.elementType | default('div') -%}
+{%- set _elementInnerWrapType = _elementType == 'div' ? 'div' : 'span' -%}
+{%- set _hasValidationStateIcon = props.hasValidationStateIcon | default(false) -%}
 {%- set _id = props.id | default(null) -%}
 {%- set _unsafeValidationText = props.UNSAFE_validationText | default(null) -%}
 {%- set _validationState = props.validationState | default(null) -%}
 {%- set _validationText = props.validationText | default(null) -%}
+{%- set _validationIconNames = {
+    success: 'check-plain',
+    warning: 'warning',
+    danger: 'danger',
+} %}
+{%- set _validationIconName = _validationIconNames[_hasValidationStateIcon]|default(null) -%}
 
 {# Attributes #}
```

Comment:
please add a new category `{# Miscellaneous #}` after Attributes

and move here: 
* _elementInnerWrapType
* _validationIconNames
* _validationIconName

They are not API options.